public/stylesheets/styles.css

```diff
@@ -1,7 +1,136 @@
-/*
+/* 
 white: #f7f9f9;
 light-blue: #2081c3;
 dark-blue: #0a2c42;
 grey: #bfbfbf;
 font-family: Arial, Helvetica, sans-serif;
-*/
+ */
+
+* {
+    margin: 0;
+    padding: 0;
+
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #bfbfbf;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+/* Layout css */
+
+.navbar {
+    background-color: #2081c3;
+    width: 100%;
+    height: 70px;
+    position: sticky;
+    top: 0;
+    display: flex;
+    align-items: center;
+}
+
+.navbar ul {
+    display: flex;
+    justify-content: start;
+    flex-direction: row;
+    margin-left: 50px;
+
```

Comment:
Remove extra spaces here. Same thing on next lines.

Not a big deal now but starting to take the habit of writing clean and organised code will definitely help you in the long run.